jbrowse/data/Caenorhabditis elegans/trackList.json

```diff
@@ -60,7 +60,7 @@
 	 },
          "label" : "Variants",
          "type" : "CanvasVariants",
-         "urlTemplate" : "/jbrowse/VCF_WBcel235_20.vcf.gz",
+         "urlTemplate" : "/jbrowse/data/worm/VCF_WBcel235_20.vcf.gz",
```

Comment:
can you simply remove the leading slash from these so that:

`         "urlTemplate" : "/jbrowse/VCF_WBcel235_20.vcf.gz",	`

becomes 

`         "urlTemplate" : "jbrowse/VCF_WBcel235_20.vcf.gz",	`

?

Reply:
that fixes all of my code

Reply:
removing the leading slash breaks it for me :-) . What does the absolute path look like that apollo wants to use to get to the vcf files? I'm sure we can get to something we both can use.